Security/day03/Medium/Challenge-1/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-alpine
+
+# Set flag secret key
+ENV FLAG=PoC{JWT_ls_s0_Gr3At_UnT1L_1T_iS_St0R3dED_1N_DB}

```

Comment:
Flag should not be in thid repository

Reply:
delete all challenges and just give a readme to give the the url of pathwar